test/compositionGuards.spec.ts

```diff
@@ -0,0 +1,23 @@
+import {assert} from 'chai';
+import * as Chance from 'chance';
+const chance = new Chance();
+import {isDateOrStringDate} from '../src';
+
+describe('compositionGuards', function() {
+	let date: Date;
+	beforeEach(function() {
+		date = new Date();
+	});
+	it('should return true for a date', function() {
```

Comment:
Please look at the other tests in the repo for the standard way the guard tests are written.  There's a common pattern that should be kept.